eo-maven-plugin/src/main/java/org/eolang/maven/PhiTranslatorMojo.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2016-2023 Objectionary.com
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included
+ * in all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package org.eolang.maven;
+
+import org.apache.maven.plugins.annotations.LifecyclePhase;
+import org.apache.maven.plugins.annotations.Mojo;
+
+/**
+ * Read XMIR file and translate it to the phi-calculus expression.
+ *
+ * @since 0.32
+ * @todo #2536:30min We need to implement exec() method.
+ *  The main idea is to read program written in EO and translate it to Phi-calculus.
+ *  To store this result we suggest to create a new folder: xmir-to-phi.
+ *  Let's store it to this folder one file by one.
+ */
+@Mojo(
+    name = "xmir-to-phi",
+    defaultPhase = LifecyclePhase.PROCESS_SOURCES,
+    threadSafe = true
+)
+public final class PhiTranslatorMojo extends SafeMojo {
```

Comment:
@Graur I think the name of the mojo should be verb, like "parse", "optimize", "transpile"

Reply:
@maxonfjvipon Agree. `PhiTranlate` will be ok?

Reply:
@Graur maybe something like "TranslateToPhi"? Or "convert"?